### DISCUSSION TITLE
Ender 3V2 Bed Visualizer Plugin gcode from wiki does not work

### DISCUSSION THREAD

**Discussion by knuxyl** [mriscoc/Ender3V2S1#649] [Q&A]

**Comment by mriscoc** [comment]
If you downloaded the latest firmware version for the Ender 3V2 with 4.2.2 board, it has UBL included.

![image](https://user-images.githubusercontent.com/2745567/212457451-ec770793-a340-49f2-bd26-a402c7f6508d.png)

**Comment by knuxyl** [comment]


**Reply by mriscoc** [reply]
When you try to submit a bug report the system ask you for this:

1- Describe the bug
2- How to Reproduce
3- Expected behavior
4- Screenshot
5- **Version (please complete the following information):** Firmware version and release date
6- Additional context

If you indeed are using an old non-UBL or a self-compiled firmware version and the Bed Visualizer g-code script is not working for you, try increasing the value for **processing time** in the plugin settings.
